bucket/dasel.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.8.0",
+    "description": "Query and update data structures from the command line. Comparable to jq/yq but supports JSON, TOML, YAML and XML with zero runtime dependencies",
```

Comment:
mention it is shortcut for `data selector`

Reply:
Done